workers/unity/Assets/Gdk/Core/Tests/Editmode/GameObjectRepresentation/EntityGameObjectLinkerTests.cs

```diff
@@ -0,0 +1,48 @@
+using NUnit.Framework;
+using Unity.Entities;
+using UnityEngine;
+
+namespace Improbable.Gdk.Core.EditmodeTests
+{
+    [TestFixture]
+    public class EntityGameObjectLinkerTests
+    {
+        private World world;
+        private EntityGameObjectLinker entityGameObjectLinker;
+        private GameObject testGameObject;
+        private Entity testEntity;
+        private const long testSpatialEntityId = 1337;
+
+        [SetUp]
+        public void Setup()
+        {
+            world = new World("TestWorld");
+            entityGameObjectLinker = new EntityGameObjectLinker(world);
+            testGameObject = new GameObject();
+            testEntity = world.GetOrCreateManager<EntityManager>().CreateEntity();
+        }
+
+        [TearDown]
+        public void TearDown()
+        {
+            if (testGameObject != null)
+            {
+                Object.DestroyImmediate(testGameObject);
+            }
+
+            world.Dispose();
+        }
+
+        [Test]
+        public void LinkGameObjectToEntity_adds_SpatialOSComponent_component()
```

Comment:
There should also be a test for `LinkGameObjectEntity_adds_gameobject_components_to_entity()` I think.

Reply:
This would require us to make the buffered commands of the `ViewCommandBuffer` publicly available. Are you ok with that?

Reply:
Why would you need that? You can call flush on the ViewCommandBuffer and check the components on the entity?